app/src/main/java/com/khanhhuy/movie_db/model/CategoryList.java

```diff
@@ -0,0 +1,29 @@
+package com.khanhhuy.movie_db.model;
+
+import android.databinding.ObservableArrayList;
+import android.util.Log;
+import android.view.View;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by yeu_thuong on 11/6/2017.
+ */
+
+public class CategoryList {
+    public List<Category> mCategories = new ArrayList<>();
+    public static final String[] CATEGOR_NAME ={"Popular", "Now playing", "Upcoming", "Top rate"};
+
+    public CategoryList(){
```

Comment:
https://stackoverflow.com/questions/10230929/how-can-i-convert-string-to-arrayliststring